runtime/j9vm/jvm.c

```diff
@@ -1189,6 +1189,14 @@ preloadLibraries(void)
 	 * location if not preloaded. */
 	preloadLibrary(J9_HARMONY_PORT_LIBRARY_SHIM_DLL_NAME, TRUE);
 #endif /* J9VM_OPT_HARMONY */
+
+#if (defined(S390) && defined(LINUX))
+	void *zlibDLL = preloadLibrary(J9_ZIP_DLL_NAME, TRUE);
+	if (NULL == zlibDLL) {
+		fprintf(stderr,"Failed to load: %s\n", J9_ZIP_DLL_NAME);
+		exit( -1 );	/* failed */
+	}
+#endif /* defined(S390) && defined(LINUX) */
```

Comment:
It would be useful to document why this is only done on Linux on Z and why we are preloading this DLL in the first place.